members/charles_carroll.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: default
+---
+
+<!-- Replace `example_student` with your name -->
+<img src="/website/assets/img/charles_carroll.png" alt="Placeholder Image" class="center" style="max-width: 100%">
+
+<!-- Replace `Example Student` with your name and include your start date-->
+# **Charles Carroll (2023 - present)**
+
+<!-- Choose your title -- feel free to be professionally silly -->
+**Role in the Cersonsky Lab**: Graduate Student Researcher.
+
+<!-- Name at least one research topic amongst this list -->
+**Things I'm excited about right now**: Representations, chemistry, computers.
+
+<!-- Ultimately, we'll use this section to
+     include papers and talks, and contributions
+     But for now put whatever you want -->
+**Cool stuff I've done**: 
+* Dug a sidewalk.
+* BD simulations of coarse grained polymers in flow.
+
+
+<!-- If you have photos you would like to exhibit,
+     save them as `/assets/member_images/your_name_photo_#.png`
+     and replace example_student below -->
+
+|      |      |      |
+|:----:|:----:|:----:|
+|![](/website/assets/img/charles_carroll_1.png) | ![](/website/assets/img/charles_carroll_2.png) | ![](/website/assets/img/charles_carroll_3.png) | ![](/website/assets/img/charles_carroll_4.png) |
```

Comment:
Because you have four pictures, might be good to place on a 2x2 grid like this:

```suggestion
|      |      |
|:----:|:----:|
|![](/website/assets/img/charles_carroll_1.png) | ![](/website/assets/img/charles_carroll_2.png) |
|![](/website/assets/img/charles_carroll_3.png) | ![](/website/assets/img/charles_carroll_4.png) |
```